Meshtastic/Helpers/BLEManager.swift

```diff
@@ -504,7 +533,19 @@ class BLEManager: NSObject, CBPeripheralDelegate, ObservableObject {
 				case .audioApp:
 					MeshLogger.log("ℹ️ MESH PACKET received for Audio App UNHANDLED \(try! decodedInfo.packet.jsonString())")
 				case .tracerouteApp:
-					MeshLogger.log("ℹ️ MESH PACKET received for Trace Route App UNHANDLED \(try! decodedInfo.packet.jsonString())")
+					if let routingMessage = try? RouteDiscovery(serializedData: decodedInfo.packet.decoded.payload) {
+						
+						if routingMessage.route.count == 0 {
+							MeshLogger.log("🪧 Trace Route request sent to \(decodedInfo.packet.from) was recieived directly.")
+						} else {
+							
+							var routeString = "🪧 Trace Route request returned: \(decodedInfo.packet.to) --> "
+							for node in routingMessage.route {
+								routeString += "\(node) --> "
+							}
+							routeString += "\(decodedInfo.packet.from)"
```

Comment:
Nice to see it being implemented for the Apple apps. I am not familiar with Swift at all, but don't you need a "MeshLogger.log(routeString)" here as well?

Reply:
Yes, just added